mne/io/nirx/nirx.py

```diff
@@ -366,16 +410,29 @@ def prepend(li, str):
                 ch_names.extend([self.ch_names[2 * ci:2 * ci + 2]] * len(on))
 
         # Read triggers from event file
-        if op.isfile(files['hdr'][:-3] + 'evt'):
-            with _open(files['hdr'][:-3] + 'evt') as fid:
-                t = [re.findall(r'(\d+)', line) for line in fid]
-            for t_ in t:
-                binary_value = ''.join(t_[1:])[::-1]
-                trigger_frame = float(t_[0])
-                onset.append(trigger_frame / samplingrate)
-                duration.append(1.)  # No duration info stored in files
-                description.append(float(int(binary_value, 2)))
-                ch_names.append(list())
+        if is_aurora:
+            if op.isfile(files['tri']):
+                with _open(files['tri']) as fid:
+                    t = [re.findall(r'(\d+)', line) for line in fid]
+                for t_ in t:
+                    trigger_frame = float(t_[7])
+                    trigger_value = float(t_[8])
+                    onset.append(trigger_frame / samplingrate)
+                    duration.append(1.)  # No duration info stored in files
+                    description.append(trigger_value)
+                    ch_names.append(list())
+
+        else:
+            if op.isfile(files['hdr'][:-3] + 'evt'):
+                with _open(files['hdr'][:-3] + 'evt') as fid:
+                    t = [re.findall(r'(\d+)', line) for line in fid]
+                for t_ in t:
+                    binary_value = ''.join(t_[1:])[::-1]
+                    trigger_frame = float(t_[0])
+                    onset.append(trigger_frame / samplingrate)
+                    duration.append(1.)  # No duration info stored in files
+                    description.append(float(int(binary_value, 2)))
+                    ch_names.append(list())
```

Comment:
If for non-aurora you add `files['tri'] = files['hdr'][:-3] + 'evt'` above somewhere, then this can be simplified:
```suggestion
        if op.isfile(files['tri']):
            with _open(files['tri']) as fid:
                t = [re.findall(r'(\d+)', line) for line in fid]
            for t_ in t:
                if is_aurora:
                    trigger_frame = float(t_[7])
                    desc = float(t_[8])
                else:
                    desc = ''.join(t_[1:])[::-1]
                    trigger_frame = float(t_[0])
                onset.append(trigger_frame / samplingrate)
                duration.append(1.)  # No duration info stored in files
                description.append(desc)
                ch_names.append(list())
```